src/3d_agent.cpp

```diff
@@ -4640,23 +4640,23 @@ void CheckPinballBonus(
 
 	// Check TOTAL ENEMY bonus
 	//
-	if (gamestuff.level[gamestate.mapon].stats.total_enemy ==
+	if (gamestuff.level[gamestate.mapon].stats.total_enemy <=
```

Comment:
Since it's a workaround please add appropriate comment.
Like this "// FIXME #86, #176".

Reply:
Comment added as full links.